library/ssl_srv.c

```diff
@@ -2765,6 +2765,55 @@ static int ssl_write_hello_verify_request( mbedtls_ssl_context *ssl )
 }
 #endif /* MBEDTLS_SSL_DTLS_HELLO_VERIFY */
 
+static void ssl_check_id_based_session_resumption( mbedtls_ssl_context *ssl )
```

Comment:
From the description it looks like you were meaning to push follow-up commits (such as the one that changed this name) - is the current state of the PR what you intended to push?

Reply:
I squashed the follow-up improvements into the commit to keep the history simpler.

Reply:
Ok, I'm fine with squashing. I'm still not happy with the name `ssl_check_xxx` for a function that returns `void`, though :)

Reply:
Fixed.